companies/123greetings.json

```diff
@@ -1,22 +1,23 @@
 {
-    "slug": "123greetings",
+    "slug": "123greeting2s",
     "relevant-countries": [
-        "all"
+        "all",
+        "dong"
     ],
     "categories": [
         "entertainment"
     ],
     "name": "123Greetings.com Inc.",
-    "address": "255 Executive Drive, Suite 400\nPlainview New York 11803\nUnited States of America",
+    "address": "123Greetings.com Inc.\n255 Executive Drive, Suite 400\nPlainview New York 11803\nUnited States of America",
     "fax": "+1 212 202 4738",
     "email": "privacy@123greetings-inc.com",
-    "web": "https://www.123greetings.com",
+    "web": "%https://www.123greetings.com",
     "sources": [
         "https://www.123greetings.com/privacy_policy.html",
         "https://www.123greetings.com/terms_of_use.html",
         "https://github.com/datenanfragen/data/pull/1025"
     ],
-    "needs-id-document": false,
+    "needs-id-document": "ja",
```

Comment:
🚫 **[recordlint]** <[pass-schema](https://github.com/datenanfragen/data#data-formats)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record did not pass schema validation: “The value at /needs-id-document must be a boolean but it was a string.”

Raw error:
```json
{
    "path": "$.needs-id-document",
    "pointer": "/needs-id-document",
    "message": "The value at /needs-id-document must be a boolean but it was a string.",
    "data": "ja"
}
```